lib/telegraf/telegraf.conf.erb

```diff
@@ -616,6 +619,7 @@
   #
   ## Some queries we may want to run less often (such as SHOW GLOBAL VARIABLES)
   interval_slow                   = "30m"
```

Comment:
I don't think so, I didn't see anything other than the obvious `fieldpass` metrics show up in my testing.